frontend/src/modules/icons/BellNoneIcon.tsx

```diff
@@ -1,6 +1,21 @@
+/**
+ * @summary A short one-line description for the file
+ *
+ * @file More detailed description for the file, if necessary;
+ * perhaps spanning multiple lines.
+ */
 import React from 'react';
 import SvgIcon, { SvgIconProps } from '@material-ui/core/SvgIcon';
 
+/**
+ * BellNoneIcon
+ *
+ * @component some description
```

Comment:
a brief description instead of the placeholder

Reply:
Updating the description for all icons